app/graphql/types/delete_list_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Types
```

Comment:
Nit: you could DRY it by creating `DeleteItemType` that will have a field `deleted_item_id`. This type would be used for all delete cases (instead of `Types::DeleteListType` and `Types::RemoveMovieType`)